app/controllers/nonsipp/landorpropertydisposal/LandOrPropertyDisposalListController.scala

```diff
@@ -212,6 +212,13 @@ class LandOrPropertyDisposalListController @Inject()(
         numberOfDisposals >= maxPossibleNumberOfDisposals ||
         allPropertiesFullyDisposed
 
+      statusMap = for {
```

Comment:
There should be some additional filtering on the load of this controller too, to hide in progress disposals from the list.

Reply:
Yes, I missed this as it was already implemented using the Completed flag. I will update to use the new Progress flag

Reply:
sure